packages/stories/src/molecules/molecules.stories.tsx

```diff
@@ -2,46 +2,73 @@ import React from 'react';
 import { storiesOf } from '@storybook/react';
 import { action } from '@storybook/addon-actions';
 
-import { Pagination } from 'uikit';
+import { withKnobs } from '@storybook/addon-knobs';
+import { Pagination, Select, Option } from 'uikit';
 
-storiesOf('05 / Molecules|Pagination', module).add('Pagination', () => (
-  <div>
-    <h1>Pagination</h1>
+let citiesData = [
+  { name: 'London', value: 'london' },
+  { name: 'Cardiff', value: 'cardiff' },
+  { name: 'Bristol', value: 'bristol' }
+];
 
+storiesOf('05 / Molecules|Pagination', module)
+  .addDecorator(withKnobs)
+  .add('Pagination', () => (
     <div>
-      <Pagination
-        currentPage={1}
-        onPageChange={p => {
-          action(`Page changed to ${p}`);
-        }}
-        totalRecords={40}
-      />
-    </div>
+      <h1>Pagination</h1>
 
-    <br />
-    <br />
+      <div>
+        <Pagination
+          currentPage={1}
+          onPageChange={p => {
+            action(`Page changed to ${p}`);
+          }}
+          totalRecords={40}
+        />
+      </div>
 
-    <div>
-      <Pagination
-        currentPage={2}
-        onPageChange={p => {
-          action(`Page changed to ${p}`);
-        }}
-        totalRecords={80}
-      />
-    </div>
+      <br />
+      <br />
 
-    <br />
-    <br />
+      <div>
+        <Pagination
+          currentPage={2}
+          onPageChange={p => {
+            action(`Page changed to ${p}`);
+          }}
+          totalRecords={80}
+        />
+      </div>
 
-    <div>
-      <Pagination
-        currentPage={12}
-        onPageChange={p => {
-          action(`Page changed to ${p}`);
-        }}
-        totalRecords={300}
-      />
+      <br />
+      <br />
+
+      <div>
+        <Pagination
+          currentPage={12}
+          onPageChange={p => {
+            action(`Page changed to ${p}`);
+          }}
+          totalRecords={300}
+        />
+      </div>
     </div>
+  ));
+
+storiesOf('05 / Molecules|Select Input', module).add('Select Input', () => (
+  <div>
+    <h1>Select</h1>
+    <Select placeholder="Select a city">
```

Comment:
Are we ever gonna have instances where the placeholder is not just a text? But like 2-line text or with an icon or something?